core/src/main/java/io/jenkins/pluginhealth/scoring/probes/JiraOpenIssuesProbe.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 Jenkins Infra
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.jenkins.pluginhealth.scoring.probes;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.Optional;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Component
+@Order(AbstractOpenIssuesProbe.ORDER)
+class JiraOpenIssuesProbe extends AbstractOpenIssuesProbe {
+    public static final String KEY = "jira-open-issues";
+    private static final String JIRA_HOST = "https://issues.jenkins.io/rest/api/latest/search?";
+    private static final Logger LOGGER = LoggerFactory.getLogger(JiraOpenIssuesProbe.class);
+    final ObjectMapper objectMapper = new ObjectMapper();
+    HttpClient httpClient;
+    HttpRequest httpRequest;
+
+    /**
+     * Get total number of open JIRA issues in a plugin.
+     *
+     * @param context {@link ProbeContext}
+     * @return Optional of type Integer.
+     */
+    @Override
+    Optional<Integer> getCountOfOpenIssues(ProbeContext context) {
+        String viewJiraIssuesUrl = context.getIssueTrackerUrlsByNames().get("jira");
+
+        if (viewJiraIssuesUrl == null || viewJiraIssuesUrl.isEmpty()) {
+            LOGGER.info("The plugin does not use JIRA to track issues.");
```

Comment:
Same remark: which plugin?

Reply:
I don't think this was addressed.

Reply:
This requires the `plugin` to be passed as a parameter to `getCountOfOpenIssues` to get `plugin.getName()` that is why 
 did not  work on this one.

Reply:
or you could just provide the `name` of the plugin, rather than the entire `plugin`.

Reply:
> or you could just provide the `name` of the plugin, rather than the entire `plugin`.

I did something similar last time. You said we shouldn't pass a parameter just to display in the logs, But if you are cool with it I  can do that,